yt_dlp/update.py

```diff
@@ -351,7 +351,9 @@ def _version_compare(self, a: str, b: str):
         return a == b
 
     def query_update(self, *, _output=False) -> UpdateInfo | None:
-        """Fetches and returns info about the available update"""
+        """Fetches info about the available update
+        @returns   UpdateInfo() if there is an update available, else None
```

Comment:
This should describe the type, so it would be `UpdateInfo`. Same below, `Optional[UpdateInfo]` or `UpdateInfo | None`